packages/app-desktop/gui/ResizableLayout/utils/useLayoutItemSizes.ts

```diff
@@ -47,10 +54,43 @@ function calculateChildrenSizes(item: LayoutItem, parent: LayoutItem | null, siz
 		}
 
 		sizes[child.key] = { width: w, height: h };
+
 		if (w !== null) remainingSize.width -= w;
 		if (h !== null) remainingSize.height -= h;
-		if (w === null) noWidthChildren.push({ item: child, parent: item });
-		if (h === null) noHeightChildren.push({ item: child, parent: item });
+		if (w === null) {
+			noWidthChildren.push({ item: child, parent: item });
+			noWidthChildrenMinWidth += child.minWidth || itemMinWidth;
```

Comment:
That implies that minWidth and minHeight cannot be zero, which is reasonable I guess. If that's the behaviour you want, then please add a test unit for it.